README.md

```diff
@@ -1,220 +1,300 @@
-### Deploying Redis Enterprise K8s using an operator (custom controller)
-### Table of Contents
+# Overview
```

Comment:
Why not just create a GKE-MARKETPLACE-README.md?
Is there a specification on repo structure?

Reply:
@ushachar my goal was to create a new RedisLabs:GKE_Marketplace branch, if it doesn't make sense, maybe we can have a new repo for that?
cc: @amiramm